test/Integration/Mapper/Strategy/IfElseTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace ScriptFUSIONTest\Integration\Mapper\Strategy;
+
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use ScriptFUSION\Mapper\InvalidConditionException;
+use ScriptFUSION\Mapper\Mapper;
+use ScriptFUSION\Mapper\Strategy\IfElse;
+
+final class IfElseTest extends \PHPUnit_Framework_TestCase
+{
+    use MockeryPHPUnitIntegration;
```

Comment:
It seems to me Mockery is not used in this test, in which case this trait is not needed and should be removed.